tests/test_type_serialization.py

```diff
@@ -1,16 +1,48 @@
-from base64 import b64decode, b64encode
-
+from base64 import b64encode
+from pathlib import Path
 
+import pytest
 from ollama._types import Image
+import tempfile
 
 
-def test_image_serialization():
-  # Test bytes serialization
+def test_image_serialization_bytes():
   image_bytes = b'test image bytes'
+  encoded_string = b64encode(image_bytes).decode()
   img = Image(value=image_bytes)
-  assert img.model_dump() == b64encode(image_bytes).decode()
+  assert img.model_dump() == encoded_string
+
 
-  # Test base64 string serialization
+def test_image_serialization_base64_string():
   b64_str = 'dGVzdCBiYXNlNjQgc3RyaW5n'
   img = Image(value=b64_str)
-  assert img.model_dump() == b64decode(b64_str).decode()
+  assert img.model_dump() == b64_str  # Should return as-is if valid base64
+
+
+def test_image_serialization_plain_string():
+  img = Image(value='not a path or base64')
+  assert img.model_dump() == 'not a path or base64'  # Should return as-is
```

Comment:
Currently just a passthrough. Since spaces are valid b64 I'm hesitant to split on it and check the length. although we could do a heuristic to catch it greedily